src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,118 @@
+
+main {
+  display: flex;
+  justify-content: center;
+  align-content: stretch;
+}
+
+.catalog {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: (4 * $width-card) + (3 * $gap-card);
+  margin-top: 50px;
+  margin-bottom: 72px;
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    width: $width-card;
+    margin-right: $gap-card;
+    margin-bottom: $gap-card - 2px;
```

Comment:
Avoid using magic numbers. Consider using a variable for the bottom margin adjustment, e.g., $card-bottom-margin-adjustment: 2px;